web/core/src/jsMain/kotlin/org/jetbrains/compose/web/elements/InputElements.kt

```diff
@@ -13,54 +13,86 @@ private fun InputAttrsBuilder<String>.applyAttrsWithStringValue(
     attrs()
 }
 
+/**
+ * It's a controlled Input of [InputType.Checkbox].
```

Comment:
I suggest us to link to the relevant piece of code where we explain what controlled actually means.

Reply:
improved